### ISSUE TITLE
There's no way to run with test specs specified inside a custom config file

### ISSUE THREAD

**Issue by robfreundlich** [mbolotov/intellij-cypress#97]
We've got a bunch of custom cypress config files set up to make it easy to run subsets of our tests from the command line. Each one has a dependency on the main config and then has something like this:

```
  "testFiles": [
    "test/cypress/integration/path/to/some/test/TheTest.spec.ts",
    "test/cypress/integration/a/different/path/to/some/other/tests/**/*.spec.ts"
  ],
```

I can specify this confg file in the test runner like so:

![image](https://user-images.githubusercontent.com/7975013/175555279-78cfc431-f8a8-434f-95cd-a97171467fdb.png)

But then when I try to run, the runner complains that I don't have a spec file. Or a test, if I click on "Test". Or a directory, if I click on "All in directory".

It'd be great if there were a fourth option, something like "Tests specified in config file". The run config could either provide a spot for the config file (per-configuration), or let the user fill it in in the program arguments, as I've done here.

**Comment by mbolotov** [comment]
Hi Rob!
Could you try this EAP bundle with the feature you requested?
[Cypress Support Pro-3.3-EAP.zip](https://github.com/mbolotov/intellij-cypress/files/9383238/Cypress.Support.Pro-3.3-EAP.zip)
Please report here any issues you find.

**Comment by mbolotov** [comment]
implemented in 3.3 version
